tests/glhe/borehole/test_base.py

```diff
@@ -1,10 +1,15 @@
 import unittest
 
 from glhe.borehole.base import BoreholeBase
+from glhe.borehole.simpleHX import BoreholeSimpleHX
```

Comment:
Added the SimpleHX class so we could actually test both the base and the Simple one.  Eventually I'm assuming we want to split those, but this got the unit test passing.

Reply:
Agreed. It will need to be built out and tested eventually.